pymc/distributions/multivariate.py

```diff
@@ -1463,6 +1482,28 @@ def helper_deterministics(cls, n, packed_chol):
         corr = inv_stds[None, :] * cov * inv_stds[:, None]
         return chol, corr, stds
 
+    @classmethod
+    def _make_packed_coord_from_dims(cls, n, dims, name_prefix, lower=True, k=0):
+        mod = pm.modelcontext(None)
+        chol_dims = [mod.coords[dim] for dim in dims]
+        if lower:
+            f_idx = np.tril_indices
+        else:
+            f_idx = np.triu_indices
+
+        flat_tri_idx = np.arange(n**2, dtype=int).reshape(n, n)[f_idx(n, k=k)]
+        coord_product = np.fromiter([f"{x}" for x in product(*chol_dims)], dtype="object")
+        tri_coords = coord_product[flat_tri_idx].tolist()
+
+        packed_dim_name = f"{name_prefix}_{dims[0]}"
```

Comment:
I would much prefer a postfix instead of a prefix, especially in this case, because we already use that for the deterministics.